gdnative-core/src/lib.rs

```diff
@@ -56,7 +55,7 @@ pub mod private;
 //
 
 pub use init::{InitializeInfo, TerminateInfo};
-pub use new_ref::NewRef;
```

Comment:
This is a good change. It's probably a good idea to keep all the similar stuff together, as such merging `new_ref` and `object` is a really good change.

Reply:
Yes, my idea is that all helper types related to object and reference handling would become part of the `object` module. I also moved `object.rs` to `object/mod.rs`, which contains the large part of the symbols.